src/main/java/com/revlearn/team1/dto/discussionBoard/DiscussionBoardFullDTO.java

```diff
@@ -0,0 +1,9 @@
+package com.revlearn.team1.dto.discussionBoard;
+
+import com.revlearn.team1.model.Course;
+
+public record DiscussionBoardFullDTO(
+        Long discussionBoardId,
+        Course course,
```

Comment:
I'd recommend removing the Course reference and making it course ID. I THINK that would be the same as your Light DTO, which is why I'm not sure the purpose of both.

The reason we want to use CourseID instead of Course is, whenever Michael changes course, you're going to have to update your entire API request which is a pain in the ass

Reply:
Sorry, had to go ask myself why I did this.
Full -> request 
Light -> response
I was getting an infinite loop when I send a response, probably because of a table relationship with Course. The Light DTO was my solution.
Also, to be honest, was trying to avoid more typing in DiscussionBoardMapper.